notebooks/notebook_enhancements/NotebookWidgetsTraining.ipynb

```diff
@@ -0,0 +1,611 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>Working with Jupyter Widgets</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "<hr style=\"height:2px;\">\n",
+    "\n",
+    "In this notebook, we'll explore several interactive [Jupyter Widgets](https://ipywidgets.readthedocs.io/en/stable/user_guide.html) and use them to incorporate user input to make dynamic, customizable plots.\n",
+    "<div style=\"float:right; width:100 px\"><img src=\"../../instructors/images/widgets_preview.png\" alt=\"example of customizable plot with widgets\" style=\"height: 250px;\"></div>\n",
```

Comment:
It looks like this preview image didn't get added to the PR

Reply:
=(
Should be fixed now

Reply:
Heck yeah, that image is great!